app/templates/events.hbs

```diff
@@ -1 +1,21 @@
+<div class = "ui grid container">
```

Comment:
No `container` required here. We already have everything wrapped by a container in `application.hbs`